drivers/volume/portworx/portworx.go

```diff
@@ -72,7 +73,7 @@ import (
 const (
 
 	// default API port
-	defaultAPIPort = 9001
+	defaultAPIPort = 9023
```

Comment:
why  the change of default rest api for PX ? If it's for tls you will need to make changes in `BuildClientsEndpoints` method to recognise this for from px-api service

Reply:
Endpoints change is here: https://github.com/libopenstorage/openstorage/pull/2092/files

I've rolled this back, it was a holdover from the old PR I picked up. I'll deal with cluster pairs in a different PR/ticket PWX-26101

`defaultAPIPort` is only used for CreatePair if not overridden, how is this port used in a clusterPair? I don't know much about the connectoin between 2 clusters.